### PR TITLE
DB-11610/DB-11267 fix sysvw.SYSCOLPERMSVIEW, add more SYSVW views

### PR DESCRIPTION
**Pull request by martinrupp**
## Short Description
Fixing issues with SYSVW.SYSCOLPERMSVIEW, SYSCONGLOMERATES, SYSDEPENDS, SYSSEQUENCES

### DB-11610
Fix NPE when performing `select * from SYSVW.SYSCOLPERMSVIEW`.
Reason was an overlooked null check in another code change and the use of a UserType in the view (changed this to a CAST to LONG VARCHAR).

### DB-11267
Currently some SYS.XXX tables can not be accessed from JDBC shells due to a marshalling error because some columns are UserTypes. Since we use those tables internally, we can't just change the column types, but we can creates views that cast those types on server side to char/varchar.

- sys.SYSCONGLOMERATES   -> SYSVW.SYSCONGLOMERATES
- sys.SYSDEPENDS         -> SYSVW.SYSDEPENDS
- sys.SYSSEQUENCES       -> SYSVW.SYSSEQUENCES

Both need an upgrade script so we combined this into one PR.

## How to test
DB-11610: Execute `select * from SYSVW.SYSCOLPERMSVIEW` from a sqlshell not in spliceengine directory.
DB-11267: Check the added views.

### CODE REVIEW

**Review by ascend1 — APPROVED** [review]
+1 once Arnaud's commnet is addressed. We need an upgrade script to reflect the view def string change.

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/2592/

File: splice_machine/src/test/java/com/splicemachine/derby/test/framework/SpliceUnitTest.java [hunk 1]
```diff
@@ -1267,4 +1267,25 @@ protected void runOltpOlapOlapNativeSpark(ThrowingBiConsumer<Boolean, Boolean> c
...
+        String[] o2 = in.split("\n");
+        String[] ex2 = expectedOutRegex.split("\n");
+        Assert.assertEquals(in + "\n---\n" + expectedOutRegex, o2.length, ex2.length);
+        for(int i =0; i<o2.length; i++) {
+            Assert.assertTrue("\n" + o2[i] + "\n--------------\ndoesn't match\n--------------\n" + ex2[i], o2[i].matches(ex2[i]));
+        }
+    }
+    public static String escapeRegexp(String asterixFilter)
+    {
+        String filter = asterixFilter;
```
Comment (arnaud-splice) [inline splice_machine/src/test/java/com/splicemachine/derby/test/framework/SpliceUnitTest.java:1282]: asterisk*

**Review by arnaud-splice — APPROVED** [review]

File: splice_machine/src/test/java/com/splicemachine/derby/test/framework/SpliceUnitTest.java (hunk 1 above)
Comment (martinrupp) [reply splice_machine/src/test/java/com/splicemachine/derby/test/framework/SpliceUnitTest.java]: done

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/2595/

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/2597/

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Review by carolp-503 — APPROVED** [review]
+1

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/2604/

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

**Pull request merged**